site/source/docs/porting/Audio.rst

```diff
@@ -0,0 +1,143 @@
+.. _Audio:
+
+=====
+Audio
+=====
+
+Emscripten supports OpenAL 1.1 "out of the box", with no additional compilation flag required. The implementation uses the Web Audio API.
```

Comment:
While this has been true for OpenAL, we actually want to move away from this model and use the same linker directive that native systems use to link to OpenAL. That is, use `-lopenal` to link to OpenAL which I believe Linux systems do(?) The PR #4665 made this directive available, with the intent of making that mandatory in the future. It would be good to document that possibility here as the "desired" mechanism.